sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-10-08)
```

Comment:
What commits/code changes are included as part of this release (between beta.1 and 2)?

Reply:
I picked up changes since the last release on Sept 7th.  I didn't add mentions to the EngSys or non-public changes

Reply:
@ahsonkhan waiting on your review

Reply:
Done. Thanks for the reminder.